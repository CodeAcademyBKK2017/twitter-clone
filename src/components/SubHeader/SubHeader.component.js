import subHeaderStyles from './SubHeader.style.css';
import React, {Component} from 'react';
class SubHeader extends  Component {
  render () {
    return (
      <div className={subHeaderStyles.subHeader} > 
        <div className={`${subHeaderStyles.container} ${subHeaderStyles.initial}`} >
          <div className={subHeaderStyles.titile}> TWEETS</div>
          <div className={subHeaderStyles.value}> 200 </div>
        </div>
        <div className={`${subHeaderStyles.container} ${subHeaderStyles.buttomHover}`} >
          <div className={subHeaderStyles.titile}> PHOTOS/VIDEOS</div>
          <div className={subHeaderStyles.value}> 200 </div>
        </div>
        <div className={`${subHeaderStyles.container} ${subHeaderStyles.buttomHover}`} >
          <div className={subHeaderStyles.titile}> FOLLOWING</div>
          <div className={subHeaderStyles.value}> 200 </div>
        </div>
        <div className={`${subHeaderStyles.container} ${subHeaderStyles.buttomHover}`} >
          <div className={subHeaderStyles.titile}> FOLLOWERS</div>
          <div className={subHeaderStyles.value}> 1M </div>
        </div>
      </div>
    );
  }
}
export default SubHeader;