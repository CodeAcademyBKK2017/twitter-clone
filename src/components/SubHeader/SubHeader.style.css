.subHeader{
        background-color: white;
        box-shadow: 0 2px 2px 0 rgba(0,0,0,0.08), 0 0 0 1px rgba(0,0,0,0.04);
        display: flex;
        align-content: flex-start;
        width:100%;
}
.container{
    display: flex;
    justify-content:flex-start;
    align-items: center;      
    flex-direction: column;
    background-color: white;
    padding: 1vh;
    flex :2;
}
.titile{
    display:flex;
color: #cbccce   
}
.value{
    display:flex;
    color: hsl(220, 3%, 20%)   
}
.initial{
    border-bottom:  2px solid #1da1f2;
}
.buttomHover{
 
}
.buttomHover:hover{
    border-bottom:  2px solid #cbccce   ;
}
