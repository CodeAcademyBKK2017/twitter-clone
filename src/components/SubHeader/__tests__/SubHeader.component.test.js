import SubHeader from '../SubHeader.component';
import React from 'react';
import renderer from 'react-test-renderer';
describe('SubHeader group Test', () => {
  it('SubHeader: snapshot test', () => {
    const snapshot = renderer.create(<SubHeader />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});