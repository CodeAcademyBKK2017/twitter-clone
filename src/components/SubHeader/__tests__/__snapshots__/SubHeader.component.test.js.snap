// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader group Test SubHeader: snapshot test 1`] = `
<div
  className={undefined}
>
  <div
    className="undefined undefined"
  >
    <div
      className={undefined}
    >
       TWEETS
    </div>
    <div
      className={undefined}
    >
       200 
    </div>
  </div>
  <div
    className="undefined undefined"
  >
    <div
      className={undefined}
    >
       PHOTOS/VIDEOS
    </div>
    <div
      className={undefined}
    >
       200 
    </div>
  </div>
  <div
    className="undefined undefined"
  >
    <div
      className={undefined}
    >
       FOLLOWING
    </div>
    <div
      className={undefined}
    >
       200 
    </div>
  </div>
  <div
    className="undefined undefined"
  >
    <div
      className={undefined}
    >
       FOLLOWERS
    </div>
    <div
      className={undefined}
    >
       1M 
    </div>
  </div>
</div>
`;
